crates/hyperswitch_domain_models/src/router_request_types.rs

```diff
@@ -279,6 +279,7 @@ pub struct PaymentsPreProcessingData {
     pub mandate_id: Option<api_models::payments::MandateIds>,
     pub related_transaction_id: Option<String>,
     pub redirect_response: Option<CompleteAuthorizeRedirectResponse>,
+    pub metadata: Option<serde_json::Value>,
```

Comment:
Should we consider keeping this as a `Secret` to avoid exposing any sensitive information, and since this is unstructured data?

CC: @Narayanbhat166 @jarnura